assets/css/general.css

```diff
@@ -223,12 +146,24 @@ html[dir='rtl'] .clearButton {
     z-index: 10000000000000;
 }
 
-/* RTL checking */
+.fa.float-right { /* sass-lint:disable-line class-name-format */
+    margin-left: .3em;
+}
 
-html[dir='rtl'] .pull-right:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
+/* RTL */
+
+html[dir='rtl'] body {
+    text-align: right !important; /* sass-lint:disable-line no-important */
+}
+
+html[dir='rtl'] .float-right { /* sass-lint:disable-line class-name-format */
     float: left !important; /* sass-lint:disable-line no-important */
 }
 
-html[dir='rtl'] .pull-left:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
+html[dir='rtl'] .float-left { /* sass-lint:disable-line class-name-format */
     float: right !important; /* sass-lint:disable-line no-important */
 }
+
+html[dir='rtl'] .close {
```

Comment:
Is this referencing the translation interface button? If so, I think we can move this rule.

Reply:
Nope, They are footer models' closing buttons. I think its general.